.github/workflows/deploy.yml

```diff
@@ -0,0 +1,50 @@
+name: "Deploy to gh-pages when PR is merged"
+on:
+  pull_request:
+    # Deploy after merging PRs to master
+    types: closed
+    branches:
+      - master
+  schedule:
+    # Deploy hourly between 9am and 7pm on weekdays
+    - cron: "0 9-19 * * 1-5"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set commit message up front
+        id: commit_message_writer
+        run: | # `github.event.number` will be blank if this is a cron
+          if [ "${{ github.event.number }}" == "" ]; then
+            echo "::set-output name=commit_message::Hourly scheduled redeploy"
+          else
+            echo "::set-output name=commit_message::Deploy via merge"
+          fi
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
```

Comment:
This repo doesn't have submodules, so I don't think we need this.

Reply:
Agreed. I'll do some experimenting to see if I can remove or change this.
(I think it was needed at one point because it was missing a local `gh-pages` branch)

Reply:
It wasn't needed 👍 now removed.